script/build.js

```diff
@@ -422,7 +422,12 @@ if (options.watch) {
     historyApiFallback: {
       rewrites: [
         { from: '^/track-claims(.*)', to: '/track-claims/' },
-        { from: '^/education/apply-for-education-benefits/application(.*)', to: '/education/apply-for-education-benefits/application/' },
+        { from: '^/education/apply-for-education-benefits/application/1990(\/.*)?', to: '/education/apply-for-education-benefits/application/1990/' }, // eslint-disable-line no-useless-escape
+        { from: '^/education/apply-for-education-benefits/application/1990E(.*)', to: '/education/apply-for-education-benefits/application/1990E/' },
```

Comment:
These appear to be case-sensitive, so I think it should be `^/education/apply-for-education-benefits/application/1990[eE](.*)` (similar change for the 1990N).

Reply:
Great catch and idea @jbalboni, changes made.